pkg/columns/formatter/json/json.go

```diff
@@ -208,34 +207,73 @@ func (f *Formatter[T]) getPrinter(level int, cols []*column[T]) func(entry *T, b
 		}
 		handlePrefix()
 
-		funcs = append(funcs, func(entry *T, buf *encodeState) {
+		funcs = append(funcs, func(indent string, entry *T, buf *encodeState) {
 			col.formatter(buf, entry)
 		})
 	}
 
-	return func(entry *T, buf *encodeState) {
+	return func(indent string, entry *T, buf *encodeState) {
 		for _, f := range funcs {
-			f(entry, buf)
+			f(indent, entry, buf)
 		}
 	}
 }
 
+func (f *Formatter[T]) formatEntry(indent string, entry *T, buf *encodeState) {
+	if entry == nil {
+		buf.WriteString(indent + "{}")
```

Comment:
This is wrong, it should be `null` (without quotation marks); was also wrong for `FormatEntry()` before, where it was an empty string instead.